src/PowerShellEditorServices/Session/RemoteFileManager.cs

```diff
@@ -74,16 +74,22 @@ public class RemoteFileManager
 
             Register-EngineEvent -SourceIdentifier PSESRemoteSessionOpenFile {0}
```

Comment:
Sorry to pollute your PR, but I found another one.  The input for `{0}` is passed on [line 494](https://github.com/tylerl0706/PowerShellEditorServices/blob/efaf12963465ca97d6c63e34141146fa6687d79c/src/PowerShellEditorServices/Session/RemoteFileManager.cs#L494).  It either passes `string.Empty` for local runspaces or `-Forward` for remote runspaces.  `Register-EngineEvent` requires either the `-Forward` switch or the `-Action` script block parameter. 

```powershell
Register-EngineEvent -SourceIdentifier PSESRemoteSessionOpenFile
# Register-EngineEvent : Action must be specified for non-forwarded events.
# At line:1 char:1
# + Register-EngineEvent -SourceIdentifier PSESRemoteSessionOpenFile
# + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#     + CategoryInfo          : InvalidArgument: (:) [Register-EngineEvent], ArgumentException
#     + FullyQualifiedErrorId : ACTION_MANDATORY_FOR_LOCAL,Microsoft.PowerShell.Commands.RegisterEngineEventCommand
```
I'm thinking we probably just need to always pass `-Forward`.  This may explain `psedit` not working properly with some session types.

Reply:
> This may explain psedit not working properly with some session types.

which session types? I haven't had any issue with WinRM, SSH, or PowerShell Direct

Reply:
That said, I can change:
```
Register-EngineEvent -SourceIdentifier PSESRemoteSessionOpenFile {0} 
```
to
```
Register-EngineEvent -SourceIdentifier PSESRemoteSessionOpenFile -Forward
```

Reply:
@tylerl0706 ah right, the condition is probably never hit.  It should still be changed at some point, but I doubt it actually affects anything.

Reply:
`-Forward` added :)